src/cellink/util/signalhandler.cpp

```diff
@@ -102,30 +110,27 @@ void SignalHandler::setup(std::function<void()> func)
     callback = func;
 
 #ifdef Q_OS_UNIX
-    const struct Signal {
+    const struct Signal
+    {
         int num;
         sighandler_t handler;
-    } handlers[] = {
-        { SIGHUP, sigHupHandler },
-        { SIGINT, sigIntHandler },
-        { SIGTERM, sigTermHandler }
-    };
+    } handlers[] = {{SIGHUP, sigHupHandler}, {SIGINT, sigIntHandler}, {SIGTERM, sigTermHandler}};
 
     for (const Signal sig : handlers) {
         struct sigaction act;
         act.sa_handler = sig.handler;
         sigemptyset(&act.sa_mask);
         act.sa_flags |= SA_RESTART;
         if (sigaction(sig.num, &act, 0))
-           qWarning() << "SignalHandler: failed to register" << sig.num;
+            qWarning() << "SignalHandler: failed to register" << sig.num;
```

Comment:
Can we force clang-format to add curly braces on all ifs?

Reply:
yes, we should. I should've checked for that.

Reply:
It does not support that unfortunately. We have to use clang-tidy or clazy. :(